src/lua/funcs/lua_registry.rs

```diff
@@ -1,21 +1,21 @@
 //! Metamethods for accesssing the registry values!
 
-use super::super::{json_to_lua, lua_to_json};
+use super::super::super::convert::{ToTable, FromTable};
 use super::super::super::registry;
 use registry::*;
 
 use hlua::any::AnyLuaValue;
+use hlua::any::AnyLuaValue::*;
 
-use rustc_serialize::json::{Json, ToJson};
 use std::ops::Deref;
 
 pub fn index(table: AnyLuaValue, lua_key: AnyLuaValue) -> AnyLuaValue {
-    if let AnyLuaValue::LuaString(key) = lua_key {
-        let maybe_json = registry::get_json(&key);
-        if let Some(json_pair) = maybe_json {
-            let (access, json) = json_pair;
+    if let LuaString(key) = lua_key {
+        let maybe_lua = registry::get_lua(&key);
+        if let Some(lua_pair) = maybe_lua {
```

Comment:
Since you only use `maybe_lua` here, maybe it would be cleaner to just do `if let Some(lua_pair) = registry::get_lua(&key) {`. Less mental overhead when you don't have to remember another variable.